src/ralph/cli.py

```diff
@@ -570,7 +585,32 @@ def convert(from_, to_, ignore_errors, fail_on_unknown, **conversion_set_kwargs)
     default=None,
     help="Query object as a JSON string (database and HTTP backends ONLY)",
 )
-def read(backend, archive, chunk_size, target, query, **options):
+@click.option(
+    "-r",
+    "--raw-output",
+    is_flag=True,
+    show_default=True,
+    default=True,
+    help="Yield bytes instead of dictionaries.",
+)
```

Comment:
In the cli context, is there a difference between `raw_output` set to `True` or `False`?

Reply:
Indeed, no different in the cli context, we can safely remove the option

Reply:
And set `raw_output` to `False` when calling the `read` backend method